### ISSUE TITLE
Start Menu Shortcuts Doesn't Work with WSL2 + X410

### ISSUE THREAD

**Issue by validatedev** [WhitewaterFoundry/Pengwin#546]
**Describe the bug**
They are created successfully but doesn't give any response.
For example, GIMP's shortcut has the following:
`C:\Windows\System32\wscript.exe C:\\Users\\valid\wslu\runHidden.vbs C:\\Users\\valid\\AppData\\Local\\Microsoft\\WindowsApps\\WhitewaterFoundryLtd.Co.16571368D6CFF_kd1vv0z0vy70w\\pengwin.exe run "cd ~; export DISPLAY=:0;  gimp-2.10"`
I think there's a problem with `export DISPLAY=:0`.

**To Reproduce**
1. Create start menu shortcuts from pengwin-setup.
2. Try to execute one of the programs' shortcut that has been created.

**Expected behavior**
It should be executable without any problems.

**Basic Troubleshooting Checklist**

[.] I have searched Google for the error message.
[.] I have checked official WSL troubleshooting documentation: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.
[.] I have searched the official Microsoft WSL issues page: https://github.com/Microsoft/WSL/issues.
[.] I have searched the Pengwin issues page: https://github.com/WhitewaterFoundry/Pengwin/issues.
[.] I have reset Pengwin: Settings->Apps->Apps & features->Pengwin->Advanced Options->Reset. 
[.] I have disabled and re-enabled WSL in Windows Features.
[.] I have run Windows 10 updates and restarted.

What other troubleshooting have you attempted?
I tried to change `export DISPLAY=:0` with `export DISPLAY=$(cat /etc/resolv.conf | grep nameserver | awk '{print $2; exit;}'):0.0` but I can't write that long on there.

**Pengwin Version**

Find: Settings->Apps->Apps & features->Pengwin->Advanced Options->Version.

Insert here: 1.2.8.0

**Windows Build**

Run 'systeminfo | findstr /C:"OS"' in Command Prompt and insert here:

OS Name: Microsoft Windows 10 Pro
OS Version: 10.0.19025 N/A Build 19025
OS Manufacturer: Microsoft Corporation
OS Configuration: Standalone Workstation
OS Build Type: Multiprocessor Free
BIOS Version: HUAWEI 1.19, 12/08/2019

For help on retrieving: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#check-your-build-number

**Comment by crramirez** [comment]
Yes, this is indeed the problem. Thank you for reporting. We are working hard to bring as soon as possible full support for WSL2

**Comment by branchard** [comment]
You have to enable 'Allow Public Access' option in X410
![image](https://user-images.githubusercontent.com/6990948/73936924-280e7180-48e4-11ea-8c9e-a5b319209785.png)
then you have to allow X410 to comunicate through the Windows firewall, it should work now.
